### PR TITLE
PUBDEV-3267  

### PR DESCRIPTION
**Pull request by st-pasha**
Prevent server from dying when encountering a list of strings; prevent client from ever sending it in the first place (hopefully)

---
This change is [Reviewable](https://reviewable.io/reviews/h2oai/h2o-3/116)

### CODE REVIEW

File: h2o-py/h2o/frame.py [hunk 1]
```diff
@@ -30,9 +32,8 @@
 from h2o.utils.shared_utils import (_handle_numpy_array, _handle_pandas_data_frame, _handle_python_dicts,
                                     _handle_python_lists, _is_list, _is_str_list, _py_tmp_key, _quoted,
                                     can_use_pandas, quote)
-from h2o.utils.typechecks import I, U, assert_is_type, assert_satisfies, is_type, numpy_ndarray, pandas_dataframe
-
-import requests
+from h2o.utils.typechecks import (assert_is_type, assert_satisfies, I, is_type, numeric, numpy_ndarray,
+                                  pandas_dataframe, U)
```
Comment (mmalohlava) [inline h2o-py/h2o/frame.py]: Minor comment: what is I, U?

**Comment by mmalohlava** [comment]
:+1:

**Comment by nkarpov** [comment]
Reviewed 3 of 3 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/h2oai/h2o-3/116)*

File: h2o-py/h2o/frame.py (hunk 1 above)
Comment (st-pasha) [inline h2o-py/h2o/frame.py]: U = union of types, I = intersection of types (really it ought to have been "∩", but you can't have a variable name like that)

**Pull request merged**